src/components/LocationDetails.js

```diff
@@ -1,22 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/LocationDetails.css";
-
-function LocationDetails(props) {
-  const { city, country, errorMessage } = props;
-  return errorMessage ? (
-    <h1>{errorMessage}</h1>
-  ) : (
-    <h1 className="location-details">{`${city}, ${country}`}</h1>
-  );
-} /* I think the ? after error Message (efter the return) means return the error message if the error message exists */
```

Comment:
you're right! remember a ternary looks t one condition and returns the left hand side if true, or the right hand side if false so what you are saying is:
if there is an errorMessage, then it will be truthy which means I want to render the error message in this h1
if there is no errorMessage, it will be an empty string which is falsy so render a h1 with the city and the country
Don't forget to ask the cohort channel or one of the tutors if you want something explained again. It's what we're here for